pkg/cmd/env/list.go

```diff
@@ -0,0 +1,34 @@
+package env
+
+import (
+	"github.com/coreeng/corectl/pkg/cmdutil/config"
+	corectlenv "github.com/coreeng/corectl/pkg/env"
+	"github.com/coreeng/developer-platform/pkg/environment"
+	"github.com/spf13/cobra"
+)
+
+func listCmd(cfg *config.Config) *cobra.Command {
+	listCmd := &cobra.Command{
+		Use:   "list",
+		Short: "List all environments",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return list(cfg)
+		},
+	}
+	return listCmd
+}
+
+func list(cfg *config.Config) error {
+	existing, err := environment.List(environment.DirFromCPlatformRepoPath(cfg.Repositories.CPlatform.Value))
+	if err != nil {
+		return err
+	}
+
+	table := corectlenv.NewTable("Cloud Platform", "Cluster", "ID")
```

Comment:
The middle one is Cloud Account (GCP Project or AWS Account ID) rather than Cluster.

The ID i think we could call Name